gadgets/snapshot_process/README.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Using snapshot process'
+weight: 20
+description: >
+  Gather information about running processes.
+---
+
+> ⚠️ This feature is experimental and could change without prior notification. Check the installation guide to enable [experimental features](../../docs/getting-started/install-linux.md#experimental-features).
```

Comment:
Since this doc aims to be published on artifacthub and all gadgets published on artifacthub are necessarily image-based gadgets, I suggest to remove that line: this image-based gadget is not more experimental than other image-based gadgets.

What do you think?

Reply:
> What do you think?

The main reason to include it here is to ensure the user doesn't get any surprises with `kubectl gadget run snapshot_process` like they need to deploy Inspektor Gadget with experimental to be able to follow steps on this guide?

Reply:
Or perhaps we should rephrase it to say `This feature requires Inspektor Gadget has been installed with experimental features enabled`